### PR TITLE
[AC-1895] AC Team code ownership moves: Bitwarden Portal

### PR DESCRIPTION
**Pull request by eliykat**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [x] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Move Organization and Provider files in the `Admin` project under AC Team ownership.

## Code changes

* **file.ext:** Description of what was changed and why

## Before you submit

- Please check for formatting errors (`dotnet format --verify-no-changes`) (required)
- If making database changes - make sure you also update Entity Framework queries and/or migrations
- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

File: src/Admin/AdminConsole/Views/_ViewStart.cshtml
Comment (eliykat) [inline src/Admin/AdminConsole/Views/_ViewStart.cshtml:1]: Stolen from Auth Team's code ownership move. This tells all views in this folder and subfolders to use the default layout without having to repeat it at the start of every view.

File: src/Admin/AdminConsole/Views/_ViewImports.cshtml
Comment (eliykat) [inline src/Admin/AdminConsole/Views/_ViewImports.cshtml:1]: Provides default imports for all views in this folder and subfolders.

File: src/Admin/Startup.cs [hunk 1]
```diff
@@ -107,6 +107,7 @@ public void ConfigureServices(IServiceCollection services)
         services.Configure<RazorViewEngineOptions>(o =>
          {
              o.ViewLocationFormats.Add("/Auth/Views/{1}/{0}.cshtml");
+             o.ViewLocationFormats.Add("/AdminConsole/Views/{1}/{0}.cshtml");
```
Comment (eliykat) [inline src/Admin/Startup.cs:110]: Tells MVC where to find views in our team-oriented folder structure.

**Review by eliykat — COMMENTED** [review]
Points of interest that are MVC/Razor specific.

**Comment by addisonbeck** [comment]
Seems straight forward. Github reported conflicts, so I pushed a merge commit so you could hopefully merge this today.

I think you could probable use [`ViewLocationExpander`s](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.razor.razorviewengineoptions.viewlocationexpanders?view=aspnetcore-8.0) to avoid using full paths for those partial views, but frankly I don't think it's worth it.

**Comment by eliykat** [comment]
Thanks! Yes I saw some suggestions around that but decided to leave it for now.

I've requested some QA on this before merging because some of these changes are only evaluated at runtime (e.g. PartialAsync), so worth having QA click through it first.

**Pull request merged**